doc/reference/tooling/tt_cli/configuration.rst

```diff
@@ -69,6 +71,13 @@ app section
     Default: ``var/lib``.
 *   ``vinyl_dir`` -- the directory where vinyl files or subdirectories are stored.
     Default: ``var/lib``.
+
+    .. note::
+
+        In all directories specified in ``*_dir`` parameters, ``tt`` creates separate
+        directories for each instance in the environment. Their names match the instances'
```

Comment:
Not sure if this is important. Separate directories for applications. Each app directory has separate sub-dirs for instances.

Reply:
Clarified this in the text.